React/Views/ScrollView/RCTScrollView.m

```diff
@@ -577,6 +577,23 @@ - (void)scrollToOffset:(CGPoint)offset
 - (void)scrollToOffset:(CGPoint)offset animated:(BOOL)animated
 {
   if (!CGPointEqualToPoint(_scrollView.contentOffset, offset)) {
+  BOOL isHorizontal = [self isHorizontal:_scrollView];
+  if (isHorizontal) {
```

Comment:
Why should we have this condition?
Seems we just have to have one single expression which ensure that `offset` inside allowed range.

Reply:
Updated PR to handle offset range in single expression.